src/System.Runtime.Numerics/tests/BigInteger/PerformanceTests.cs

```diff
@@ -21,7 +21,7 @@ public PerformanceTests(ITestOutputHelper output)
         }
 
         [Theory]
-        [ActiveIssue("PerformanceTest")]
```

Comment:
OK @Jiayili1 to summarize, for the PerformanceTest ones, they should be just 
```
[Theory]
[Benchmark]
```

and no ActiveIssue of course.